css/portal-dashboard/header.less

```diff
@@ -8,7 +8,6 @@
   box-sizing: border-box;
   height: @header-height;
   margin-bottom: 3px;
-  min-width: 1100px;
```

Comment:
If I remember correctly, I added these `min-width` values so if your browser window was narrow, you could still horizontally scroll over to view content that wasn't visible. With these changes you can't do that anymore. Any content on the right side of the dashboard that's not visible in the browser window becomes inaccessible unless you can resize the browser.

Was it decided that that's OK though?

Reply:
I agree.  I think we should clarify with Kiley and Trudi what they want changed/fixed.  I posted in the Slack thread to get clarification.

Reply:
We definitely want to fix that overlap bug, but I think that is just a couple lines of CSS which could be extracted into a much simpler PR.